### PR TITLE
Describe `node.outputCount` for function nodes

### PR DESCRIPTION
**Pull request by kristianheljas**
Implementing PR: https://github.com/node-red/node-red/pull/2918

Not yet published, but merged - should be available in version 1.3.

### CODE REVIEW

File: docs/user-guide/writing-functions.md [hunk 1]
```diff
@@ -80,6 +80,29 @@ var newMsg = { payload: msg.payload.length };
...
+const messages = new Array(node.outputCount)
+// Choose random output number to send the message to
+const chosenOutputIndex = Math.floor(Math.random() * node.outputCount);
+// Send the message only to chosen output
+messages[chosenOutputIndex] = msg;
+// Return the array containing chosen output
+return messages;
+{% endhighlight %}
+
+You can now configure
```
Comment (knolleary) [inline docs/user-guide/writing-functions.md:104]: Incomplete sentence?

File: docs/user-guide/writing-functions.md [hunk 2]
```diff
@@ -80,6 +80,29 @@ var newMsg = { payload: msg.payload.length };
 return [msg, newMsg];
 {% endhighlight %}
 
+#### Handling arbituary number of outputs
+
+*Since Node-RED 1.3*
+
+`node.outputCount` now contains the number of the outputs configured for the function node.
```
Comment (knolleary) [inline docs/user-guide/writing-functions.md:87]:
```suggestion
`node.outputCount` contains the number of the outputs configured for the function node.
```

**Review by knolleary — CHANGES_REQUESTED** [review]
Looks good - a couple things to tidy

**Comment by kristianheljas** [comment]
Whoopsie, forgot to hit save before commit.

Should be good now :+1:

**Comment by kristianheljas** [comment]
Idk, does "Changes requested" not go away when force pushing?

**Review by knolleary — APPROVED** [review]

**Comment by knolleary** [comment]
Thanks

**Pull request merged**